versioned_docs/version-1.3.0/install-kubeslice/kubeslice-cli/install-kubeslice.mdx

```diff
@@ -13,8 +13,27 @@ controller cluster and the worker cluster names. The file should include other r
 as described in the sample configuration file provided below. This file is essential in setting 
 up KubeSlice.
 
-Custom chart values can be specified in the topology configuration. For example, you can add `values:metrics.insecure=true` under 
+
+### Air-Gapped Installation
+
+To install KubeSlice in Air-Gapped environment use local helm charts. Upgrade to the latest 
+[kubeslice-cli version 0.6.0](https://github.com/kubeslice/kubeslice-cli/releases) to use local charts.
+
+Make sure you set the **use_local** value to `true` in the topology file:  
```

Comment:
use_local also can be enclosed in backticks as it's a property name.

Reply:
I had set it to bold to highlight the parameter